addons/nuxeo-platform-3d/nuxeo-platform-3d-core/src/main/java/org/nuxeo/ecm/platform/threed/importer/ThreeDImporter.java

```diff
@@ -82,7 +82,7 @@ protected String getModelFilename(final Blob zipContent) throws IOException {
         try (ZipInputStream zipInputStream = new ZipInputStream(zipContent.getStream())) {
             while ((zipEntry = zipInputStream.getNextEntry()) != null) {
                 // skip if the entry is a directory, if it's not a supported extension or if it's hidden (by convention)
-                boolean isSupported = SUPPORTED_EXTENSIONS.contains(FileUtils.getFileExtension(zipEntry.getName()));
+                boolean isSupported = SUPPORTED_EXTENSIONS.contains(lowerCaseFileExtension(zipEntry.getName()));
```

Comment:
What about using:
```java
String ext = FilenameUtils.getExtension(zipEntry.getName).toLowerCase();
boolean isSupported = SUPPORTED_EXTENSIONS.contains(ext);
```

Reply:
You mean  ```org.apache.commons.io.FilenameUtils.getExtension() ```  insteadof
``` org.nuxeo.common.utils.FileUtils.getFileExtension() ```  ? 
(yes we could)

Reply:
instead of the new `lowerCaseFileExtension` method.